CONTRIBUTING.md

```diff
@@ -107,6 +107,18 @@ RECAPTCHA_KEY=REPLACE_ME
 
 4. Visit [`http://local.artsy.net:5000`](http://local.artsy.net:5000).
 
+<details>
+   <summary>Do you see an error about not providing a secure connection?</summary>
+
+Your browser has probably cached a redirect from `http://*.artsy.net` to `https://...`. Clear your browser cache for this page: 
+
+- Browse to https://local.artsy.net:5000
+- Open Chrome Dev Tools (this adds a drop down menu to the reload icon)
+- Click and hold “Reload” icon until the drop down appears.
+- Select “Empty Cache and Hard Reload”
+- You may now browse to http://local.artsy.net:5000
+</details>
```

Comment:
One (possible) fix would be to [only mount this](https://github.com/artsy/force/blob/e842a51820e365149525abb1b3c0592061044e04/src/lib/middleware/ensureSsl.ts#L12) in development: 
  https://github.com/artsy/force/blob/master/src/common-middleware.ts#L53